front/src/actions/employeeActions.jsx

```diff
@@ -1,26 +1,2 @@
-import alt from '../alt'
-import EmployeeSource from '../sources/employeeSource'
 import { List } from 'immutable'
 
```

Comment:
This is just a placeholder for now, it's never going to contain any jsx spesific stuff, so the file extension could be just .js (this is a bigger topic, should these files be .js, .jsx, .es6, .es or something else? Should they be coherent throughout the project or should we use different extensions for react components compared to ecmascript compatible code.) Not going to address this further in this pull request.